_episodes/07-errors.md

```diff
@@ -394,10 +394,10 @@ One reason for receiving this error is that you specified an incorrect path to t
 For example,
 if I am currently in a folder called `myproject`,
 and I have a file in `myproject/writing/myfile.txt`,
-but I try to just open `myfile.txt`,
+but I try to open `myfile.txt`,
 this will fail.
 The correct path would be `writing/myfile.txt`.
-It is also possible (like with `NameError`) that you just made a typo.
+It is also possible (like with `NameError`) that you made a typo.
```

Comment:
I would suggest replacing "you made a typo" with "a typo has occurred" to minimise what could be interpreted as accusation.

Reply:
In this context, "all" feels  similar to "just" in that it implies that it's easy -- which it might not be for a learner.